assets/scss/hero.scss

```diff
@@ -82,3 +129,24 @@
     }
   }
 }
+
+.exhibition-page {
+  h1 {
+    font-size: 2.375rem;
+    margin-top: 2.5rem;
+  }
+  p.lead {
+    font-size: 1.5rem;
+    color: $slategrey;
```

Comment:
Would you mind checking with the designers if there should be a new grey? See one of the other comments on greys too.

Reply:
changed to $mediumgrey